src/Libraries/Microsoft.Extensions.AI/ChatCompletion/FunctionInvokingChatClient.cs

```diff
@@ -325,7 +325,7 @@ public override async IAsyncEnumerable<StreamingChatCompletionUpdate> CompleteSt
                     // If there were any, remove them from the update. We do this before yielding the update so
                     // that we're not modifying an instance already provided back to the caller.
                     int addedFccs = functionCallContents.Count - preFccCount;
-                    if (addedFccs > preFccCount)
+                    if (addedFccs > 0)
```

Comment:
Yeah, this is just a bug. If memory serves, I originally had `if (functionCallContents.Count > preFccCount)`, then realized I needed the difference, separated it out into a local, but missed fixing up the rhs.